### ISSUE TITLE
Ability to add custom CSS or JS

### ISSUE THREAD

**Issue by fadeouter** [Jujuyeh/webapp-creator#27]
Some websites, like Tumblr, hasn't mobile layout for small screens. Ability to add custom CSS code could fix this.
Also, CSS injection useful for ad blocking purposes.

**Comment by Jujuyeh** [comment]
We will discuss this. 😄

I think uAdBlock already blocks adds from the browser and webapps.

**Comment by fadeouter** [comment]
Adblocking isn't main purpose of this feature. uAdBlock cannot remove DOM elements. CSS injecting can.
Main purpose of this is to add ability to use more websites as a real apps. As I mentioned early, not all websites have mobile layout.

**Comment by mateosalta** [comment]
Yes this is possible, I've done a userstyle.js on the youtube web app - we could add the relevant code to the ogra container and do a file selection for the js. Should work well enough if you already have something in mind to insert.
